app/controllers/hiring_staff/sign_in/dfe/sessions_controller.rb

```diff
@@ -17,6 +17,11 @@ def create
     perform_dfe_sign_in_authorisation
   end
 
+  def destroy
```

Comment:
If you put a `binding.pry` in you will see that this method is never run, correct?